jwst/tests_nightly/general/nircam/test_nircam_steps_single.py

```diff
@@ -186,3 +189,31 @@ def test_wfs_combine2(self):
                     'test_wfscomb_do_ref.fits')
                   ]
         self.compare_outputs(outputs)
+
+
+@pytest.mark.bigdata
+class TestNrcGrismSetPointing(BaseJWSTTest):
+    input_loc = 'nircam'
+    ref_loc = ['test_pointing', 'truth']
+    test_dir = 'test_pointing'
+    rtol = 0.000001
+
+    def test_nircam_setpointing(self):
+        """
+        Regression test of the set_telescope_pointing script on a level-1b NIRCam file.
+        """
+
+        # Copy original version of file to test file, which will get overwritten by test
+        input_file = self.get_data(self.test_dir,
+                                   'jw00721012001_03103_00001-seg001_nrcalong_uncal_orig.fits')
+
+        # Get SIAF PRD database file
+        siaf_prd_loc = ['jwst-pipeline', self.env, 'common', 'prd.db']
+        siaf_path = get_bigdata(*siaf_prd_loc)
+
+        add_wcs(input_file, siaf_path=siaf_path,
+                engdb_url='http://twjwdmsemwebag.stsci.edu/JWDMSEngFqAccSide1/TlmMnemonicDataSrv.svc/')
```

Comment:
Is there a reason this is different from our default URL currently in the code?

https://github.com/spacetelescope/jwst/blob/d448b27c9cc34f5ea70fb19181e890fd660f99b3/jwst/lib/engdb_tools.py#L22

Reply:
This is the url I got from the most recent DMS Ops processing logs. When I try to use the default from engdb_tools, it fails to connect. The combinations that seem to work include:
ENGDB_HOST = 'http://twjwdmsemweb.stsci.edu/' or 'twjwdmsemwebag.stsci.edu/'
but with the second level path being 'JWDMSEngFqAccSide1'. Without the substring 'Side1' it fails.

Reply:
Perhaps we should update the default in engdb_tools.

Reply:
Latest commit updates the default URL and uses it (instead of listing it as an explicit argument).